modules/update-service-graph-data.adoc

```diff
@@ -0,0 +1,31 @@
+[id="update-service-graph-data_{context}"]
+= Creating the OpenShift Update Service graph data container image
+
+The OpenShift Update Service requires a graph-data container image, from which the OpenShift Update Service retrieves information about channel membership and blocked update edges. Graph data is typically fetched directly from the upgrade graph data repository. In environments where an internet connection is unavailable, loading this information from an init container is another way to make the graph data available to the OpenShift Update Service. The role of the init container is to provide a local copy of the graph data, and during pod initialization, the init container copies the data to a volume accessible by the service.
+
+.Procedure
+
+. Create a docker file, for example, `./Dockerfile`, containing the following:
++
+[source,terminal]
+----
+FROM registry.access.redhat.com/ubi8/ubi:8.1
+
+RUN curl -L -o cincinnati-graph-data.tar.gz https://github.com/openshift/cincinnati-graph-data/archive/master.tar.gz
+
+CMD exec /bin/bash -c "tar xvzf cincinnati-graph-data.tar.gz -C /var/lib/cincinnati/graph-data/ --strip-components=1"
+----
+
```

Comment:
Above Docker file seems not workable. I just have a tested with above dockerfile to build a graph-data image, it failed to start as following:
```
# ./oc get po
NAME                                     READY   STATUS                  RESTARTS   AGE
update-68cc44f479-bm2pc                  0/2     Init:CrashLoopBackOff   7          11m
updateservice-operator-5685848f8-vm8bc   1/1     Running                 0          18m
  Normal   Pulled          <invalid> (x3 over <invalid>)  kubelet            Container image "jliu-46.mirror-registry.qe.gcp.devcluster.openshift.com:5000/openshift/graph-data:latest" already present on machine
  Normal   Created         <invalid> (x3 over <invalid>)  kubelet            Created container graph-data
  Normal   Started         <invalid> (x3 over <invalid>)  kubelet            Started container graph-data
  Warning  BackOff         <invalid> (x2 over <invalid>)  kubelet            Back-off restarting failed container
```
After I changed to another graph-data image built from [dockerfile](https://github.com/openshift/cincinnati-operator/blob/master/docs/disconnected-cincinnati-operator.md#create-the-graph-data-for-cincinnati), it works correctly.
```
# ./oc get po
NAME                                     READY   STATUS    RESTARTS   AGE
update-75bcb6f7b6-7fhw5                  2/2     Running   0          3m52s
updateservice-operator-5685848f8-vm8bc   1/1     Running   0          38m
```

Reply:
@jottofar There seems not much more logs from the instance pod. 
```
# ./oc logs update-68cc44f479-gf5rr -c graph-builder 
Error from server (BadRequest): container "graph-builder" in pod "update-68cc44f479-gf5rr" is waiting to start: PodInitializing
```
I pasted more info here https://privatebin-it-iso.int.open.paas.redhat.com/?70494bb9d993b659#9vPT8Wgqhb9RSnW2Gb9BCS4E5aBLQtTV9bwEz8CDPBek
And almost the same test steps with another graph-data image built from [dockerfile](https://github.com/openshift/cincinnati-operator/blob/master/docs/disconnected-cincinnati-operator.md#create-the-graph-data-for-cincinnati), then it works. Please help confirm what's wrong here, thx!

Reply:
@jottofar I had a check again today and found that it's my wrong dockerfile. The steps work well in this pr. So this issue should be no issue now.
```
CMD exec /bin/bash -c "tar xvzf /tmp/cincinnati-graph-data.tar.gz -C /var/lib/cincinnati/graph-data/ --strip-components=1"

```